client/globals.d.ts

```diff
@@ -10,10 +16,21 @@ declare const wcpaySettings: {
 	isJetpackConnected: boolean;
 	isJetpackIdcActive: boolean;
 	accountStatus: {
+		email?: string;
 		error?: boolean;
 		status?: string;
 		country?: string;
+		paymentsEnabled?: boolean;
+		deposits?: Array< any >;
+		depositsStatus?: string;
+		currentDeadline?: bigint;
+		pastDue?: boolean;
+		accountLink: string;
 		hasSubmittedVatData?: boolean;
+		requirements?: {
+			currentlyDue?: Array< string >;
+			errors?: Array< RequirementError >;
+		};
```

Comment:
As we return fallback values for those properties from the PHP side. We should make them non-optional by removing the `?`.